index.js

```diff
@@ -339,6 +344,31 @@ function addTest(testTitle, testCaseId, buildUrl) {
   }
 }
 
+function getAttachment(attachmentFolder, testTitle) {
+  const attachmentsFolder = path.join(process.cwd(), `${attachmentFolder}`)
```

Comment:
I think attachmentFolder can be declared once in the top.
Can't see any reason to pass attachmentFolder in current implementation

Reply:
Not sure, it's a property of qTestConfig